Dockerfile

```diff
@@ -44,6 +44,20 @@ RUN \
     mv docker /usr/bin/$DOCKER_VERSION && \
     chmod +x /usr/bin/$DOCKER_VERSION
 
+# Add new AWS RDS CA certificates
+# For details see:
+#  - https://aws.amazon.com/blogs/database/amazon-rds-customers-update-your-ssl-tls-certificates-by-february-5-2020/
+RUN \
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/gh/codacy/base/pullRequest?prid=4799033)